pkg/dynamicmetric/dynamic_exposure.go

```diff
@@ -0,0 +1,86 @@
+package dynamicmetric
```

Comment:
move to the risk package? exposure is a kind of risk control

Reply:
can't do that. there will be an import cycle

Reply:
which part is re-imported?

Reply:
moved them to risk/dynamicrisk package